ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -190,9 +186,18 @@ UIGestureRecognizerDelegate {
     URLProtocol.registerClass(KeymanURLProtocol.self)
 
     if !isSystemKeyboard {
-      copyUserDefaultsToSharedContainer()
-      copyKeymanFilesToSharedContainer()
-      let userData = activeUserDefaults()
+      if let shared = Storage.shared,
```

Comment:
Should we be copying here? If a groupID is set, the app should always be using the shared container. This code will only help migrate data when an app chooses to use a shared container where they previously did not.